qtconsole/rich_jupyter_widget.py

```diff
@@ -212,19 +213,30 @@ def _is_latex_math(self, latex):
                 return True
         return False
 
+    def _get_colors(self, color):
+        """Get colors from the current syntax style if loadable"""
```

Comment:
```suggestion
    def _get_color(self, color):
        """Get color from the current syntax style if loadable."""
```

This is getting a single color from the style, not multiple colors. So the method name is not correct.